File: app/src/main/java/omniex/nl/omniex/ui/app/profile/ProfileFragment.java
package omniex.nl.omniex.ui.app.profile;

import android.support.design.widget.TabLayout;
import android.support.v4.view.ViewPager;

import org.androidannotations.annotations.AfterViews;
import org.androidannotations.annotations.EFragment;
import org.androidannotations.annotations.ViewById;

import nl.omniex.omniexshopping.R;
import nl.omniex.omniexshopping.ui.adapters.ProfileViewPagerAdapter;
import nl.omniex.omniexshopping.ui.base.BaseFragment;

@EFragment(R.layout.fragment_profile)
public class ProfileFragment extends BaseFragment<ProfileView, ProfilePresenter> implements ProfileView {

    @ViewById(R.id.profile_vp)
    ViewPager mProfileVp;

    @ViewById(R.id.profile_tab_layout)
    TabLayout mProfileTabLayout;

    @AfterViews
    void initTabs(){
        mProfileVp.setAdapter(new ProfileViewPagerAdapter(getFragmentManager()));
        mProfileTabLayout.setupWithViewPager(mProfileVp);
    }
}

File: app/src/main/java/omniex/nl/omniex/ui/app/profile/ProfileFragment.kt
package omniex.nl.omniex.ui.app.profile

import android.support.design.widget.TabLayout
import android.support.v4.view.ViewPager
import omniex.nl.omniex.R
import omniex.nl.omniex.ui.adapters.ProfileViewPagerAdapter
import omniex.nl.omniex.ui.base.BaseFragment

import org.androidannotations.annotations.AfterViews
import org.androidannotations.annotations.EFragment
import org.androidannotations.annotations.ViewById


@EFragment(R.layout.fragment_profile)
open class ProfileFragment : BaseFragment<ProfileView, ProfilePresenter>(), ProfileView {

    @ViewById(R.id.profile_vp)
    lateinit var mProfileVp: ViewPager

    @ViewById(R.id.profile_tab_layout)
    lateinit var mProfileTabLayout: TabLayout

    @AfterViews
    internal fun initTabs() {
        mProfileVp!!.adapter = ProfileViewPagerAdapter(fragmentManager!!)
        mProfileTabLayout!!.setupWithViewPager(mProfileVp)
    }
}
